internal/backend/remote-state/azure/backend.go

```diff
@@ -214,6 +214,12 @@ func New(enc encryption.StateEncryption) backend.Backend {
 				DefaultFunc: schema.MultiEnvDefaultFunc([]string{"ARM_OIDC_REQUEST_URL", "ACTIONS_ID_TOKEN_REQUEST_URL"}, ""),
```

Comment:
You'll want to add an environment variable here and in the `oidc_request_token` below, too.